.github/workflows/ci.yml

```diff
@@ -689,6 +689,7 @@ jobs:
       - name: Test Template
         run: |
           cd templates/default
+          pnpm compile
```

Comment:
seems odd to me this had not happened before - perhaps a smell that we aught to get turbo repo working